docs/api/ref/Channel.md

```diff
@@ -36,6 +36,7 @@ interface Channel {
   broadcast(context: Context): Promise<void>;
   getCurrentContext(contextType?: string): Promise<Context|null>;
   addContextListener(contextType: string | null, handler: ContextHandler): Promise<Listener>;
+  clearContext(contextType?: string): Promise<void>;
```

Comment:
I'm wondering if we should require an explicit null argument (`contextType: string | null`) to clear all types - which would match `addContectListener` or stick with this (matching `getCurrentContext`). Either works of course.

Reply:
To me it makes sense to keep it closer to getCurrentContext, because they seem most related to me, but happy to hear any thoughts/suggestions.

Reply:
There is a historical reason for addCOntextListener having an explicit null argument, the type argument was originally optional so the `handler` argument could be either first or second (when a type was specified). This was good in practice so we moved to an explicit `null` argument and deprecated the single argument override:
https://fdc3.finos.org/docs/2.0/api/ref/DesktopAgent#addcontextlistener-deprecated

We don't have that problem here, so I agree its fine as is (matching `getCurrentContext`).